vault/diagnose/tls_verification_test.go

```diff
@@ -117,7 +117,7 @@ func TestTLSExpiredCert(t *testing.T) {
 	if !strings.Contains(errs[0].Error(), "certificate has expired or is not yet valid") {
 		t.Fatalf("Bad error message: %s", errs[0])
 	}
-	if warnings == nil || len(warnings) != 1 {
```

Comment:
Can you add a check for the other warning thrown in this test to ensure it is expected?